src/test/com/gd/twitteranalytics/TransformTweetsTest.scala

```diff
@@ -0,0 +1,59 @@
+package com.gd.twitteranalytics
+
+import java.sql.Timestamp
+import java.util.Date
+
+import com.holdenkarau.spark.testing.{DataFrameSuiteBase, StreamingSuiteBase}
+import org.scalatest.FreeSpec
+import twitter4j.Status
+import org.scalamock.scalatest.MockFactory
+
+@SerialVersionUID(100L)
+class TransformTweetsTest extends FreeSpec with Serializable with StreamingSuiteBase with MockFactory  with DataFrameSuiteBase {
```

Comment:
Is there a real need to use `StreamingSuiteBase` in the test spec?

Reply:
"testOperation" which compares input value and expected value, in first test case is function of StreamingSuitBase.

Reply:
oh, I see. thanks for the clarification.